doc/Makefile

```diff
@@ -5,13 +5,13 @@ endif
 
 all:
 	# Only build the docs if this is a git tree
-	test ! '(' -d ../.git -o -d ../.bzr ')' || $(MAKE) -C user-guide
+	-test ! '(' -d ../.git -o -d ../.bzr ')' || $(MAKE) -C user-guide
```

Comment:
This change probably means that the travis build will no longer catch breakage in the documentation build. Even if it does, other package builds can now accidentally skip the docs if this is the case.

Wouldn't a configure option (--doc=0) be better for this?